Package.resolved

```diff
@@ -226,6 +235,15 @@
           "version": "1.0.0-alpha.7"
         }
       },
+      {
+        "package": "TwitterVapor",
+        "repositoryURL": "https://github.com/handya/TwitterVapor.git",
+        "state": {
+          "branch": null,
+          "revision": "a33b9e5d832a02e269c67d54a9fabdc8d2bde855",
+          "version": "1.1.0"
+        }
+      },
```

Comment:
I've had a look through these two packages and I'd be ok adding them. While they don't have any tests, they're quite small and understandable. I'm confident we could easily fork and maintain this ourselves should there be issues - plus losing the Twitter firehose isn't really critical.

I think even if we rolled our own here it'd probably be a good idea to make these separate packages, so from that point of view it's fine. Might add some tests and upstream them.

Reply:
PS: We should definitely add them to some package index for discovery 😆

Reply:
On second thought, I think we should drop `TwitterVapor`. It requires us to route though `Application` when all we really need is a `Client` if we send the post ourselves (which isn't all that complicated). We've got the client in `reconcileVersions` where we'd fire off the request and the credentials will be accessible via `Current`.

I'll take a look later today/tomorrow!